gatsby/content/blog/2022/07/2022-07-20-synapse-1.63.mdx

```diff
@@ -0,0 +1,69 @@
+---
+date: '2022-07-20'
+title: Synapse 1.63 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+Hey all, it's time for another Synapse release! That's right, [Synapse
+1.63](https://github.com/matrix-org/synapse/releases/tag/v1.63.0) is out,
+let's have a look at it.
+
+## Clarifications on "anonimised" server statistics
+
+Synapse has the ability to report usage statistics to the Matrix.org Foundation
+(or to another location, if configured to do so). These statistics, such as
+number of users, number of rooms joined by the server, etc. (they don't feature
+any identifiable information about users and rooms) help us monitor the health
+of the public federation.
+
+In this release of Synapse, we have updated our public documentation about this
+feature to clarify how it works and what exactly is being reported. This
+documentation can be found [right
+here](https://matrix-org.github.io/synapse/v1.63/usage/administration/monitoring/reporting_homeserver_usage_statistics.html).
+
+Note that previous documentation and prompts surrounding this feature called it
+"anonimised" server statistics. This could easily be misinterpreted, as while
+per-user statistics are not reported - homeserver server names are. We have
+therefore changed said documentation and prompts to not call these statistics
+"anonimised" and be clearer about what is actually reported.
```

Comment:
This reads a bit awkwardly and I don't think we need to call out the loss of "anonymised" specifically.

```suggestion
therefore changed said documentation and prompts to be clearer about
what is actually reported.
```